core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -2339,7 +2347,9 @@ class SubmissionTestExecutionServiceDAO(workflowStatus: => String, workflowCost:
   }
 
   override def getCost(id: String, userInfo: UserInfo): Future[WorkflowCostBreakdown] =
-    Future.successful(WorkflowCostBreakdown(id, workflowCost, "USD", workflowStatus, Seq.empty))
+    if (abortedMap.keySet.contains(id))
+      Future(WorkflowCostBreakdown(id, workflowCost, "USD", WorkflowStatuses.Aborted.toString, Seq.empty))
+    else Future(WorkflowCostBreakdown(id, workflowCost, "USD", workflowStatus, Seq.empty))
```

Comment:
the submission monitor now calls `getCost()` for all workflows instead of calling `status()` for non-capped workflows and  `getCost()` for capped workflows. Thus, the abort-related logic in this mock's `status()` needed to be copied into this mock's `getCost()`.